store/src/java/com/zimbra/cs/mailbox/Mailbox.java

```diff
@@ -10667,13 +10674,21 @@ public String getRecipient() {
             return recipient;
         }
 
+        public Long getTimestamp() {
+            return timestamp;
+        }
+
         public void setDataSourceId(String dsId) {
             this.dsId = dsId;
         }
 
         public void setRecipient(String recipient) {
             this.recipient = recipient;
         }
+
+        public void setTimestamp(Long timestamp) {
```

Comment:
is MessageCallbackContext:: setDataSourceId still being used?

Reply:
As of this PR, yes. The next pull request will actually remove this in lieu of storing datasource IDs in message metadata.

Reply:
I actually missed setting the datasource ID in the MailItemImport class.